yarn-project/canary/src/utils.ts

```diff
@@ -47,16 +34,11 @@ export async function deployAndInitializeNonNativeL2TokenContracts(
   rollupRegistryAddress: EthAddress,
   initialBalance = 0n,
   owner = AztecAddress.ZERO,
-  underlyingERC20Address?: EthAddress
+  underlyingERC20Address?: EthAddress,
```

Comment:
Were formatting not run here before or skipped now? Seems like quite a few formatting changes.

Reply:
> Were formatting not run here before or skipped now? Seems like quite a few formatting changes.

Ah looks like formatting isnt enforced for it, but i ran the formatter for the whole repo. Ill probably turn the formatter on in another pr